lang/janet/files/makefile-janet-spawn-no-chdir.diff

```diff
@@ -0,0 +1,11 @@
+--- a/Makefile
++++ b/Makefile
+@@ -59,7 +59,7 @@ LIBJANET_LDFLAGS?=$(LDFLAGS)
+ RUN:=$(RUN)
+ 
+ 
+-COMMON_CFLAGS:=-std=c99 -Wall -Wextra -Isrc/include -Isrc/conf -fvisibility=hidden -fPIC
++COMMON_CFLAGS:=-DJANET_SPAWN_NO_CHDIR -std=c99 -Wall -Wextra -Isrc/include -Isrc/conf -fvisibility=hidden -fPIC
```

Comment:
I can propose a fix for this upstream if you'd like (they already do detection of older OS X versions for things like [`clock_gettime`](https://github.com/janet-lang/janet/blob/4e02f27eb95ea2bdc86464a42bf678f700dfe82c/src/core/util.c#L930))